src/Aquifer.Jobs/Subscribers/TranslationMessageSubscriber.cs

```diff
@@ -180,34 +180,41 @@ public async Task UpdateProjectPostTranslationActivity(
         var companyLeadUserId = project.CompanyLeadUserId
             ?? throw new InvalidOperationException($"Company Lead User ID is null for Project ID {project.Id}. This should never happen.");
 
+        // Assign all Draft Resource Content Versions in the project to the Company Lead of the Project.
         // Edge case handling: It's possible that something else acted on the resource content version status
         // and moved it out of the TranslationAiDraftComplete status between the resource finishing translation
         // and this project post-processing. If so, ignore those resources when performing assignments
         // because there might already have been a user assignment.
-        var resourceContentVersions = await _dbContext.ResourceContentVersions
+        // Note: resource contents with the New status are also included as these were not translated but still
+        // need to be assigned (for Aquiferization of English resources).
+        var resourceContentVersionsToAssign = await _dbContext.ResourceContentVersions
             .AsTracking()
             .Include(rcv => rcv.ResourceContent)
             .Where(rcv => rcv.ResourceContent.ProjectResourceContents.Any(prc => prc.Project.Id == project.Id) &&
                 rcv.IsDraft &&
-                rcv.ResourceContent.Status == ResourceContentStatus.TranslationAiDraftComplete)
+                (rcv.ResourceContent.Status == ResourceContentStatus.TranslationAiDraftComplete ||
+                    rcv.ResourceContent.Status == ResourceContentStatus.New))
             .ToListAsync(activityContext.CancellationToken);
 
         // Edge case handling: Only update resource content versions that were queued for translation (even if gracefully skipped).
         // Due to an unknown amount of time passing between translation and this post-processing it's unlikely
         // but still possible that the list is different.
-        foreach (var resourceContentVersion in resourceContentVersions
+        foreach (var resourceContentVersionToAssign in resourceContentVersionsToAssign
             .Where(rcv => dto.TranslatedProjectResourceContentIds.Contains(rcv.ResourceContentId)))
         {
             // now that ALL translations are complete for the project, assign all resources in the project to the company lead
-            resourceContentVersion.AssignedUserId = companyLeadUserId;
+            resourceContentVersionToAssign.AssignedUserId = companyLeadUserId;
```

Comment:
🤔 This should probably skip assignment (and also creating the snapshot) if the resource is already assigned to the company lead.  I'll address that and add a commit to the PR.

Reply:
Done here: https://github.com/BiblioNexusStudio/aquifer-server/pull/538/commits/2fb36af9c175775f1bde3877ff48f02799c97b30